Gemfile

```diff
@@ -79,6 +79,7 @@ gem "snmp",                           "~>1.1.0",      :require => false
 gem "uglifier",                       "~>2.7.1",      :require => false
 gem "novnc-rails",                    "~>0.2"
 gem 'spice-html5-rails'
+gem 'sys-proctable',                                  :require => false
```

Comment:
- If this is just for Mac, let's wrap this in a Mac conditional, so it doesn't get installed on the appliance (at least until we get the linux support going).
- If it's only used in the gems/pending, it should probably live in gems/pending/Gemfile.